packages/duckietown_demos_dbv2/.autogenerated.md

```diff
@@ -0,0 +1,21 @@
+<div id='duckietown_demos-autogenerated' markdown='1'>
+
+
+<!-- do not edit this file, autogenerated -->
+
+## Package information 
+
+[Link to package on Github](github:org=duckietown,repo=Software,path=70-convenience-packages/duckietown_demos,branch=master18)
+
+### Essentials {nonumber="1"}
+
+Maintainer: [Liam Paull](mailto:lpaull@mit.edu)
+
+### Description {nonumber="1"}
+
+TODO: description of `duckietown_demos`
+
+
+
+</div>
+
```

Comment:
Is that file needed ? This seems to be and old autoggenerated file that we dont use anymore. I would suggest removing it, as it contains outdated info